README.md

```diff
@@ -110,6 +126,25 @@ None.
       roles:
         - role: geerlingguy.redis
 
+Setup replication
```

Comment:
Why is it required to have 2 different plays. Cannot you not configure Ansible to have both hosts in the same play.